prolog/tokenize_numbers.pl

```diff
@@ -0,0 +1,7 @@
+:- module(tokenize_numbers, [number//1]).
+/** <module> Number support for the tokenizer
+ *
+ */
+:- use_module(library(dcg/basics), [number//1]).
+]
```

Comment:
Shouldn't this cause a syntax error?

Reply:
Not sure what you mean. The number//1 is valid.  
The close ] bizarelly doesn't appear in my code

Reply:
anyway, seems ok here